tutorial/4_hextof_workflow.ipynb

```diff
@@ -216,18 +503,17 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "plt.figure()\n",
+    "plt.figure() # if you are using interactive plots, you'll need to generate a new figure explicitly every time.\n",
     "res.mean('sampleBias').plot.line(x='energy',linewidth=3)\n",
     "res.plot.line(x='energy',linewidth=1,alpha=.5);"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "sp.dataframe"
+    "## correct offsets\n",
+    "The energy axis is now correct, but still the curves do not stack on eachother as we are not compensating for the `sampleBias`. In the same way, we can compensate the photon energy (`monocrhomatorPhotonEnergy`) and the `tofVoltage` "
```

Comment:
running this cell, I get the error:
`KeyError: "['sampleBias', 'monochromatorPhotonEnergy', 'tofVoltage'] not in dataframe!"`

Reply:
It seems you updated the flash_example_config whereas I thought the tutorial was using hextof_tutorial.yml

Reply:
I faced this problem since I am using the sed-processor 0.1.5 which doesnt yet include this change.

Reply:
Yes right.